src/simsopt/geo/surfaceobjectives.py

```diff
@@ -287,21 +287,40 @@ def dJ(self):
         return Derivative({self.surface: deriv})
 
 
-def boozer_surface_residual(surface, iota, G, biotsavart, derivatives=0):
+def boozer_surface_residual(surface, iota, G, biotsavart, derivatives=0, weight_inv_modB=False):
```

Comment:
Consider default True?

Reply:
I decided against defaulting to `True` here because the BoozerExact algorithm relies on this function, and it assumes that that the residuals are unweighted.